bin/beaker

```diff
@@ -3,7 +3,11 @@
 require 'rubygems' unless defined?(Gem)
 require 'beaker'
 
-Beaker::CLI.new.execute!
+if (Beaker::Subcommands.instance_methods(false) << :help).include? ARGV[0].to_sym
+  Beaker::Subcommands.start(ARGV)
+else
+  Beaker::CLI.new.execute!
+  puts "Beaker completed successfully, thanks."
```

Comment:
to me it looks a little noisy. so if i run beaker init, the output would be like:

Beaker configuration written to .beaker/config
Beaker completed successfully, thanks.

like the beaker init task is supposed to do one small task so to me it's like saying

HEY I DID THAT ONE TASK YOU ASKED OF ME
I'VE FINISHED DOING WHAT YOU ASKED 

it's like when a child tells you something they've done, and you don't quite acknowledge them, so they tell you again, but with different wording

to me, the "Beaker completed successfully, thanks." makes sense when we've printed a bunch of text to the screen, i.e. during a test run, and you can't be bothered having to read through the block so you do tl;dr and skip to the end to get the gist of what's happened.